cmd/snap-update-ns/main.go

```diff
@@ -77,7 +77,7 @@ func run() error {
 		return fmt.Errorf("cannot open lock file for mount namespace of snap %q: %s", snapName, err)
 	}
 	defer lock.Close()
-	if err := lock.Lock(); err != nil {
+	if err := lock.Lock(0); err != nil {
```

Comment:
Hmm. I think I'd rather have two methods (`Lock` and `TryLock` perhaps?), rather than having funky flags